osm2gtfs.py

```diff
@@ -29,11 +30,13 @@
 def main():
 
     # Load config json file
-    try:
-        config = json.load(args.config)
-    except ValueError, e:
-        print('Failed to load config json file:')
-        print(e)
+    if os.path.isfile('config.json'):
+        with open("config.json") as json_file:
+            config = load_config(json_file)
+    elif args.config is not None:
+            config = load_config(args.config)
```

Comment:
What happens here if the config file as been specified as an argument, but does not exist?